app/api-key-setup/page.tsx

```diff
@@ -0,0 +1,95 @@
+'use client';
+import React from 'react';
+import { useState } from 'react';
+import Image from 'next/image';
+import { IoMdInformationCircleOutline } from 'react-icons/io';
+
+function Page() {
+  const [tooltipVisible, setTooltipVisible] = useState(false);
+  return (
+    <section>
+      <div className="flex flex-col items-center justify-center h-screen mx-auto">
+        <div className="flex flex-col items-center justify-center w-[30rem] bg-[#C7D6E5] rounded-lg p-12 drop-shadow-lg">
+          <header className="flex flex-col items-center justify-center w-full mb-6">
+            <Image
+              src="/generics/logo.png"
+              alt="beacon logo"
+              className="w-[80px] h-[80px] drop-shadow-lg"
+              width={80}
+              height={80}
+            />
+            <div className="flex flex-col">
+              <h1 className="text-[32px] leading-none font-bold mb-0.5">
+                Bea<span className="text-[#4D6FBD]">con</span>
+              </h1>
+            </div>
+          </header>
+          <main className="w-full">
+            <div className="flex justify-center items-center gap-x-1  mb-2">
+              <h3 className="text-xl font-semibold">OpenAI API Key</h3>
+              <div
+                onMouseEnter={() => setTooltipVisible(true)}
+                onMouseLeave={() => setTooltipVisible(false)}
```

Comment:
@arwin50 You can easily simplify and remove this state by implementing the tailwind css peer modifier:
https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

Reply:
Done updating. Wow, I never knew that you could do it that way. What a new discovery, thanks for that! 😊